packages/grafana-ui/src/components/Collapse/CollapsableSection.tsx

```diff
@@ -5,20 +5,26 @@ import { Icon } from '..';
 import { GrafanaTheme2 } from '@grafana/data';
 
 export interface Props {
-  label: string;
+  label: string | ReactNode;
```

Comment:
minor nit, i think `ReactNode` includes `string`. but maybe it helps devs understand they can pass string as well? 🤔

Reply:
LOL, true. I'll fix

Reply:
Fixed!